BrowserKit/Sources/Common/Utilities/DynamicFontHelper.swift

```diff
@@ -61,4 +68,9 @@ public struct DefaultDynamicFontHelper: DynamicFontHelper {
     public static func preferredBoldFont(withTextStyle textStyle: UIFont.TextStyle, size: CGFloat) -> UIFont {
         return preferredFont(withTextStyle: textStyle, size: size, weight: .bold)
     }
+
+    public static func sizeOfString(_ string: String, usingFont font: UIFont) -> CGSize {
+        let attributes: [NSAttributedString.Key: Any] = [.font: font]
+        return string.size(withAttributes: attributes)
+    }
```

Comment:
Overall its fine but we didn't have any other option? 🤔 just curious

Reply:
This shouldn't go into the `DynamicFontHelper` - it is used in one place only.